build/document-extractor.ts

```diff
@@ -469,11 +470,22 @@ function _addSingleSpecialSection(
             infoEntry.version_added.startsWith("≤")
               ? infoEntry.version_added.slice(1)
               : infoEntry.version_added;
+          const removed =
+            typeof infoEntry.version_removed === "string" &&
+            infoEntry.version_removed.startsWith("≤")
+              ? infoEntry.version_removed.slice(1)
+              : infoEntry.version_removed;
```

Comment:
To avoid the duplication (468..472 + 473..477), could we extract this to `_normalizeVersion(version: bcd.VersionValue): bcd.VersionValue`?